react/official/list/src/pages/ViewMessage.tsx

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
 import { useState } from 'react';
```

Comment:
```suggestion
import React, { useState } from 'react';
```
It's failing because the `React` import was removed.

Reply:
@brandyscarney reason why it fails

Reply:
Ah that was accidentally removed when I was copying back and forth: https://github.com/ionic-team/starters/pull/1856/commits/5ec674c377df9767bcffcb2dd5ca5ab99b4ef892